LayoutTests/fast/rendering/render-multi-column-set-invalidate-crash.html

```diff
@@ -0,0 +1,74 @@
+<head>
```

Comment:
This is the reduced version of the test (see the original one in the radar)

I've tried a few things to reduce it further by removing the event handlers, but the crash doesn't repro with that, so if you folks have ideas please let me know :)